### PR TITLE
Support multi columns section

### PR DESCRIPTION
**Pull request by hayato-SMZ**
## What does this change?
- Added space element to "AectinProperty", transferring the role of columns.
- When building xml, "columns" in "SectionProperty" is now output as attribute "w:num".
- A multi-columns page file is output.

- Test and Snap was modified to match.

### CODE REVIEW

**Comment by bokuweb** [comment]
@hayato-SMZ Thanks for your great work. Looks good.
However, let me confirm, how should we set `columns`??
We need to write following code?

``` rust 
let mut d = Docx::new();
d.document.section_property.columns = 2;
```

If so, could please add `columns` method to `Document` struct.
It would be better if you could write the following

``` rust
let mut d = Docx::new();
d.document = d.document.columns(2);
```

**Comment by hayato-SMZ** [comment]
@bokuweb 
Thank you for your comment.
Add ```columns``` method to ```Document``` struct and  add ```columns``` method  docx-core tests.

File: docx-core/src/documents/document.rs [hunk 1]
```diff
@@ -270,7 +276,22 @@ mod tests {
...
   <w:body><w:sdt><w:sdtPr><w:rPr /></w:sdtPr><w:sdtContent><w:p w14:paraId="12345678"><w:pPr><w:rPr /></w:pPr><w:r><w:rPr /><w:fldChar w:fldCharType="begin" w:dirty="true" /><w:instrText>TOC \o &quot;1-3&quot;</w:instrText><w:fldChar w:fldCharType="separate" w:dirty="false" /></w:r></w:p><w:p w14:paraId="12345678"><w:pPr><w:rPr /></w:pPr><w:r><w:rPr /><w:fldChar w:fldCharType="end" w:dirty="false" /></w:r></w:p></w:sdtContent>
-</w:sdt><w:sectPr><w:pgSz w:w="11906" w:h="16838" /><w:pgMar w:top="1985" w:right="1701" w:bottom="1701" w:left="1701" w:header="851" w:footer="992" w:gutter="0" /><w:cols w:space="425" /><w:docGrid w:type="lines" w:linePitch="360" /></w:sectPr></w:body>
+</w:sdt><w:sectPr><w:pgSz w:w="11906" w:h="16838" /><w:pgMar w:top="1985" w:right="1701" w:bottom="1701" w:left="1701" w:header="851" w:footer="992" w:gutter="0" /><w:cols w:space="425" w:num="1" /><w:docGrid w:type="lines" w:linePitch="360" /></w:sectPr></w:body>
+</w:document>"#
+        );
+    }
+
+        #[test]
+    fn test_document_cols() {
```
Comment (bokuweb) [inline docx-core/src/documents/document.rs:285]: 👍

**Review by bokuweb — APPROVED** [review]
LGTM

**Comment by bokuweb** [comment]
@hayato-SMZ docx-rs 0.4.2 published

**Pull request merged**